cgp/individual.py

```diff
@@ -90,89 +124,51 @@ def randomize_genome(self, genome_params, rng):
         -------
         None
         """
-        self.genome = Genome(**genome_params)
-        self.genome.randomize(rng)
+        for g in self.genome:
+            g.randomize(rng)
 
-    def to_func(self):
+    def to_func(self) -> List[Callable[[List[float]], List[float]]]:
         """Return the expression represented by the individual as Callable.
 
         Returns
         ----------
-        Callable
+        Callable[[List[float]], List[float]]
         """
-        return CartesianGraph(self.genome).to_func()
+        return [CartesianGraph(g).to_func() for g in self.genome]
```

Comment:
i feel like this will add a lot of unpacking for the user, won't it? it's already suboptimal that they will need to do `f([...])[0]` for a function with a single output, now it would read `f[0]([...])[0]`. this is not very pretty. should we add automatic unpacking for the special case of a single genome, even if this seems to contradict best practice? this holds essentially for all cases where we add an additional list in this new implementation